v2/README.md

```diff
@@ -0,0 +1,42 @@
+# Gengo: a framework for building simple code generators
```

Comment:
Do we need to update the root gengo readme to say anything about v2 (i.e. https://github.com/kubernetes/gengo/blob/master/README.md)?

Reply:
good idea, will do